src/test/java/no/nav/tag/finnkandidatapi/kandidat/KandidatRepositoryTest.java

```diff
@@ -41,11 +41,35 @@ public void skal_kunne_lagre_og_hente_ut_med_null() {
         Kandidat behovTilLagring = enKandidatMedBareNull();
 
         Integer lagretId = repository.lagreKandidat(behovTilLagring);
-        Kandidat uthentetBehov = repository.hentKandidat(lagretId);
+        Kandidat uthentetBehov = repository.hentKandidat(lagretId).get();
 
         assertThat(uthentetBehov).isEqualToIgnoringGivenFields(enKandidatMedNullOgTommeSet(), "id");
     }
 
+    @Test
+    public void hentNyesteKandidat__skal_returnere_siste_registrerte_kandidat() {
+        Kandidat kandidat1 = enKandidat();
+        Kandidat kandidat2 = enKandidat();
+
+        repository.lagreKandidat(kandidat1);
+        repository.lagreKandidat(kandidat2);
+        Kandidat sisteKandidat = repository.hentNyesteKandidat(kandidat1.getFnr()).get();
+
+        assertThat(sisteKandidat).isEqualToIgnoringGivenFields(kandidat2, "id");
+    }
+
+    @Test
+    public void hentNyesteKandidat__skal_håndtere_henting_av_ikke_eksisterende_kandidat() {
```

Comment:
Disse to testene har motsatt navn (hentNyesteKandidat testes i hentKandidat-testen og motsatt)